backend/app/Services/Packaging/ShiftHandoverCalculator.php

```
@@ -0,0 +1,139 @@
+        )->count();
+
+        // WIP — open pallets (their accumulated qty) + still-unpacked good output.
+        $openPalletsQuery = $byLine(Pallet::query()->where('status', PalletStatus::Open->value));
+        $openPalletsQty = (int) $openPalletsQuery->sum('qty');
+        $openPalletsCount = (int) $openPalletsQuery->count();
+        $unpacked = max(0, $good - $packed);
+        $wipTotal = $openPalletsQty + $unpacked;
+
+        // Shipped — pallets dispatched during the window.
+        $shipped = (int) $byLine(
+            Pallet::query()
```

PHPStan (2.2.1)

[error] Call to an undefined static method App\Models\Pallet::query().


(staticMethod.notFound)